### PR TITLE
ENH: Enable testing

### PR DESCRIPTION
**Pull request by aflaxman**
Might need to change an option in github or travis to make this run automatically

### CODE REVIEW

**Comment by damianavila** [comment]
LGTM... an yes, you need to to sync the repo on the TravisCI interface and then, in the next push the things happen as usual...

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+install:
+  - conda create -n testenv --yes python=$TRAVIS_PYTHON_VERSION
+  - source activate testenv
+  - conda install --yes $DEPS
+  - python setup.py install
+
+before_install:
+  # setup virtual x
+  - "export DISPLAY=:99.0"
```
Comment (damianavila) [inline .travis.yml]: In fact I am curious about these two lines... why you are using them there? I don't think it is necessary, but maybe I am missing something...

Comment (aflaxman) [inline .travis.yml]: I included that to make some `matplotlib` stuff run without errors.  Not sure if it is still necessary.

**Comment by aflaxman** [comment]
Seems like I broke something: https://travis-ci.org/aflaxman/mplexporter/builds/70646736  And also seems like Travis CI has a new preferred approach: http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade

**Comment by damianavila** [comment]
Yes, sorry, I forgot to mention that, you should use `sudo: false` at the very beginning to use the docker-based infrastructure. 

Also you to change `export PATH=/home/travis/anaconda/bin:$PATH` to `export PATH="$HOME/miniconda/bin:$PATH"`

Sorry about this... I was finishing SciPy and I was a little bit tired :wink:

**Comment by aflaxman** [comment]
Cool, that fixed it: https://travis-ci.org/aflaxman/mplexporter/builds/70839232

Anyone mind if I merge this PR?

**Comment by damianavila** [comment]
:+1: for me

**Comment by aflaxman** [comment]
I'm going to merge, then

**Comment by aflaxman** [comment]
@jakevdp : I don't have admin access to this repo, so when you have a sec, can you turn on travis integration?  Here is the link to the admin page: https://travis-ci.org/profile/mpld3

**Comment by jakevdp** [comment]
> can you turn on travis integration?

done!

**Pull request merged**